From b3e7074599b4cfe57485f1f314da66e5e0d97af4 Mon Sep 17 00:00:00 2001
From: Ignacio Duart <iduartgomez@gmail.com>
Date: Thu, 12 Mar 2026 17:57:02 +0100
Subject: [PATCH 58/69] build: use freenet-stdlib 0.3.2 (with freenet-macros
 0.2.0)

---
 delegates/chat-delegate/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/delegates/chat-delegate/Cargo.toml b/delegates/chat-delegate/Cargo.toml
index 28b6548a..b407934c 100644
--- a/delegates/chat-delegate/Cargo.toml
+++ b/delegates/chat-delegate/Cargo.toml
@@ -9,7 +9,7 @@ freenet-main-delegate = []
 
 [dependencies]
 ciborium.workspace = true
-freenet-stdlib = { version = "0.3.1", features = ["contract"] }
+freenet-stdlib = { version = "0.3.2", features = ["contract"] }
 bs58 = { workspace = true }
 serde = { workspace = true, features = ["derive"] }
 river-core.workspace = true
-- 
2.53.0

